crates/ln-dlc-node/src/ln/event_handler.rs

```diff
@@ -244,7 +251,7 @@ where
                     counterparty,
                     funding_satoshis,
                     push_msat,
-                    "Accepting 0-conf channel request"
```

Comment:
I believe 0-conf was correct here. Although we are just implying this code is only executed for the mobile app. See also configs for the mobile app where we manually accept channel requests.